components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -62,31 +64,59 @@ function StarcraftMatchSummary.MatchSummary(props)
 		:addClass('brkts-popup')
 		:addClass('brkts-popup-sc')
 		:addClass(match.opponentMode == 'uniform' and 'brkts-popup-sc-uniform-match' or 'brkts-popup-sc-team-match')
-		:node(StarcraftMatchSummary.Header({match = match}))
+		:node(StarcraftMatchSummary.Header({match = match, config = props.config}))
 		:node(StarcraftMatchSummary.Body({match = match}))
 		:node(StarcraftMatchSummary.Footer({match = match, showHeadToHead = match.headToHead}))
 end
 
 StarcraftMatchSummary.propTypes.Header = {
 	match = StarcraftMatchGroupUtil.types.Match,
+	config = 'table?'
```

Comment:
might be simpler to set defaults for `config` and `config.showScore` near the beginning of `StarcraftMatchSummary.MatchSummary`. that way don't need to do nil checks everywhere.